cloud/services/domains.go

```diff
@@ -200,8 +198,7 @@ func DeleteDomainRecord(ctx context.Context, mscope *scope.MachineScope, hostnam
 
 	// If record is A type, verify ownership
 	if recordType != "TXT" {
-		txtRecordValueString := createSHA256HashOfString(mscope.LinodeMachine.Name)
-		isOwner, ownerErr := IsDomainRecordOwner(ctx, mscope, hostname, "owner:"+txtRecordValueString, domainID)
+		isOwner, ownerErr := IsDomainRecordOwner(ctx, mscope, hostname, target, domainID)
```

Comment:
nit: just call it `err`. If it would shadow an existing `err` variable, then the lexical scope isn't tight enough.

```suggestion
		isOwner, err := IsDomainRecordOwner(ctx, mscope, hostname, target, domainID)
```

Reply:
done